bootstrap/secret/insecure.go

```diff
@@ -245,3 +246,19 @@ func (p *InsecureProvider) GetSelfJWT() (string, error) {
 func (p *InsecureProvider) IsJWTValid(jwt string) (bool, error) {
 	return true, nil
 }
+
+func (p *InsecureProvider) HttpTransport() http.RoundTripper {
+	return http.DefaultTransport
+}
+
+func (p *InsecureProvider) SetHttpTransport(_ http.RoundTripper) {
+	//empty on purpose
+}
+
+func (p *InsecureProvider) IsZeroTrustEnabled() bool {
+	return false
+}
+
+func (p *InsecureProvider) ZeroTrustEnabled() {
```

Comment:
Match how define in the interface.

```suggestion
func (p *InsecureProvider) EnableZeroTrust() {
```

Reply:
This change wasn't made